### PR TITLE
Add keyboard press method

### PR DESCRIPTION
**Pull request by alextes**
First off, this is the first PureScript PR I've made that contains more than a line or two 🎉 . That's why I'd ask to please be liberal with picking it apart. I'm trying to learn PureScript here and pick up on idioms, and even nits are welcome! You'd be helping, not offending me 😁 .

The method implemented: [keyboard.press](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#keyboardpresskey-options).

The Page object has a keyboard property which allows sending keypresses.
This PR adds the most straightforward of the Keyboard methods, sending a
single keypress of choice.

Points to discuss:
* I don't know if it's okay to access [the keyboard property on the page object](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#pagekeyboard), it doesn't look like you can do much with the object so I couldn't come up with a reason we might want a `foreign import data Keyboard` even though it might exist.
* `KeyName` is not a great name but it's also possible to make it safe by getting the list of options now linked in a comment into the code. I'm not sure what the best way would be, or where this long list should go. If you can link me an example, I'm happy to make the required changes!
* In the test I use `unsafeEvaluateStringFunction` to get at the value in the input we use to register the effect of the keyboard event. If we can use puppeteer APIs to get at it safely, I imagine that would be nicer, but we don't have `getProperty` or `ElementHandle` yet.

Thanks for taking the time to review 🙏 .

### CODE REVIEW

**Comment by justinwoo** [comment]
this seems fine, just that "press" could be named "keyboardPress" and "Keyname" could be "KeyboardKey". definitely seems like other methods of the keyboard object might as well be implemented in this way.

i think `unsafeEvaluateStringFunction` has ended up being the only reliable function for me in puppeteer, so any other methods for interacting within pages are just extras in my own usage.

File: src/Toppoki.purs [hunk 1]
```diff
@@ -186,6 +186,22 @@ runPromiseAffE3 f a b c =  Promise.toAffE $ FU.runFn3 f a b c
 runPromiseAffE4 :: forall a b c d o. FU.Fn4 a b c d (Effect (Promise o)) -> a -> b -> c -> d -> Aff o
 runPromiseAffE4 f a b c d =  Promise.toAffE $ FU.runFn4 f a b c d
 
+-- | See [USKeyboardLayout](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/lib/USKeyboardLayout.js) for a list of all key names.
+newtype KeyName = KeyName String
+
+-- | Shortcut for `keyboard.down` and `keyboard.up`.
+press
```
Comment (justinwoo) [inline src/Toppoki.purs]: should also link to https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#keyboardpresskey-options in the comment

**Comment by alextes** [comment]
Since you pointed out:
> definitely seems like other methods of the keyboard object might as well be implemented in this way.

I went ahead and did so 😁 .

I also use the `page.$eval` functions. They let you keep the selector part in PureScript and have the function run against the selected elements in an unsafe manner similar to `unsafeEvaluateStringFunction`. I use them in the tests for the keyboard functions so I added them to this PR. If you prefer I can split the PRs and stack this one on top of the other so its tests can use `page.$eval`.

**Comment by justinwoo** [comment]
it's fine to leave them together i think. afaict this seems fine for me. should we merge this soon?

**Comment by alextes** [comment]
Ya let's do it 🙌 . I've been using my own fork but I'm rather excited to start contributing some code back to the PureScript community.

**Comment by justinwoo** [comment]
:+1:

**Pull request merged**